test/test_helper.rb

```diff
@@ -0,0 +1,6 @@
+require 'simplecov'
+SimpleCov.start
+require 'minitest'
+require 'minitest/pride'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.